app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,57 @@
+class TweetsController < ApplicationController
+#==============================tweets================================ 
+  get '/tweets' do 
+    redirect '/login' if !logged_in?
+    
+    erb :"tweets/tweets"
+  end
+#===============================new================================== 
+  get '/tweets/new' do 
+    redirect '/login' if !logged_in?
+    
+    erb :'tweets/create_tweet'
+  end
+#-------------------------------------------------------------------- 
+  post '/tweets' do 
+    redirect '/tweets/new' if empty?(params)
+    
+    Tweet.create(content: params[:content], user_id: current_user.id)
```

Comment:
You can also specify `user` directly here -- for example:

```
 Tweet.create(content: params[:content], user: current_user)
```

Reply:
ah nice!! and yep makes sense